### DISCUSSION TITLE
关于使用过程中的一些问题

### DISCUSSION THREAD

**Discussion by codersyn** [AlistGo/alist#987] [Q&A]
①这个视频pc上能播放，ios上无法播放，用的safari：https://synssr.ga/%E7%94%B5%E5%BD%B1/%E5%A5%97%E8%A3%85.mkv
②这个视频pc和手机均不能播放：https://synssr.ga/%E7%94%B5%E5%BD%B1/%E4%B8%8B%E6%B0%B4%E9%81%93%E7%BE%8E%E4%BA%BA%E9%B1%BC.avi
③flac格式的音乐在电脑上能播放，ios无法播放，只能播放mp3格式
④文件无法移动，移动某个文件时提示成功了，但是刷新后文件还在原处
以上问题建立在aria2 + rclong + onedrive 国际版的基础上

**Comment by xhofe** [comment]
不要过分依赖浏览器，浏览器不是万能的，它的解码能力很弱。
1. content-type: application/octet-stream 对于Safari来说不支持跨域且响应头不为媒体格式的视频
2. 没有支持解码avi-xvid的浏览器，至少我不知道
3. iOS浏览器不支持解码
4. 无法复现，测试没有问题。

**Reply by codersyn** [reply]
好的，我自己再测试一下，谢谢

**Comment by star2k** [comment]
> ①这个视频pc上能播放，ios上无法播放，用的safari：https://synssr.ga/%E7%94%B5%E5%BD%B1/%E5%A5%97%E8%A3%85.mkv
> 
> ②这个视频pc和手机均不能播放：https://synssr.ga/%E7%94%B5%E5%BD%B1/%E4%B8%8B%E6%B0%B4%E9%81%93%E7%BE%8E%E4%BA%BA%E9%B1%BC.avi
> 
> ③flac格式的音乐在电脑上能播放，ios无法播放，只能播放mp3格式
> 
> ④文件无法移动，移动某个文件时提示成功了，但是刷新后文件还在原处
> 
> 以上问题建立在aria2 + rclong + onedrive 国际版的基础上

我这里发现在ios浏览器上，部分MP3无法播放，电脑chrome就正常，不知怎么解决？但有些mp3却正常
